packages/opencensus-exporter-azure/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './azure-stats';
+export * from './azure-trace';
```

Comment:
Usually we keep trace and stats exporters in separate packages, not sure if you want to do the same thing here.

Reply:
We were mimicking what we saw in the StackDriver exporter where both were included in the same package.